extensions/amp-story/1.0/amp-story-open-page-attachment.js

```diff
@@ -38,13 +40,49 @@ export const buildOpenAttachmentElement = (element) =>
     </a>`;
 
 /**
- * Renders the open attachment UI affordance.
+ * @param {!Element} element
+ * @return {!Element}
+ */
+export const buildOpenInlineAttachmentElement = (element) =>
+  htmlFor(element)`
+    <a class="
+        i-amphtml-story-page-open-attachment i-amphtml-story-system-reset"
+        role="button">
```

Comment:
Since this has the `role="button"`, the `aria-label` needs to be on this element.
It could be a default string like `aria-label="Open page attachment"`.
That would need to be localized string.

This could then be overridden by the `cta-text` attribute.

Reply:
We should be careful here; "page attachment" is not a term that we expose to users, so they (likely?) won't know what this means.

We normally (thus far) have erred on the side of "swipe up" or "read more" or similar.